### PR TITLE
Rm bulletclass

### PR DESCRIPTION
**Pull request by rabiyamajeed**

### CODE REVIEW

**Comment by rabiyamajeed** [comment]
bullet class

File: bullet.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Object.cpp
```
Comment (zzakiullah) [inline bullet.cpp]: bullet.cpp lol

File: bullet.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#pragma once
+// Object.cpp
```
Comment (zzakiullah) [inline bullet.h]: bullet.h lol

**Review by zzakiullah — CHANGES_REQUESTED** [review]

**Review by zzakiullah — APPROVED** [review]
Looks good

**Review by zzakiullah — APPROVED** [review]

**Review by s-yfang — APPROVED** [review]

**Pull request merged**